packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js

```diff
@@ -227,7 +233,7 @@ export default function InspectedElementWrapper(_: Props) {
             <ButtonIcon type="error" />
           </Toggle>
         )}
-        {canToggleSuspense && (
+        {!hideToggleSuspenseAction && canToggleSuspense && (
```

Comment:
Can we make this part of the `canToggleSuspense` boolean assignment?

Reply:
Done.